README.md

```diff
@@ -16,6 +16,15 @@ This documentation presents how to install and launch TractoFlow on a local comp
 
 If you are a user and NOT A DEVELOPER, we HIGHLY RECOMMEND following the instructions on the documentation website.
 
+
+Support:
+--------
+
+For questions or help on TractoFlow, please create a topic on Neurostar with the `tractoflow` tag. [https://neurostars.org](https://neurostars.org)
```

Comment:
You should create the link directly for the `Neurostar` word (in `on Neurostars`). Also, missing a `s` for the first `Neurostar`.